django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -39,6 +46,40 @@ def check_permissions(self, request: Request) -> None:
             raise PermissionDenied()
 
 
+class CyberstormTeamCreateRequestSerialiazer(serializers.Serializer):
+    name = serializers.CharField(
+        max_length=Team._meta.get_field("name").max_length,
+        validators=Team._meta.get_field("name").validators,
+    )
+
+
+class CyberstormTeamCreateResponseSerialiazer(serializers.Serializer):
+    name = serializers.CharField()
+
+
+class TeamCreateAPIView(APIView):
```

Comment:
Missing permission classes (user needs to be authenticated)

Reply:
All of the new views should now have `IsAuthenticated`